updates_and_signals/safe_message_handlers/workflow.py

```diff
@@ -229,9 +196,7 @@ def should_continue_as_new(self) -> bool:
     async def run(self, input: ClusterManagerInput) -> ClusterManagerResult:
         self.init(input)
         await workflow.wait_condition(lambda: self.state.cluster_started)
-        # Perform health checks at intervals.
         while True:
```

Comment:
There is no value in this being a loop anymore and no value in the wait condition having a timeout (and therefore no value in a sleep interval setting)

Reply:
Ah, very good point! (Fixed same bug in Typescript version). I think this actually improves the sample a lot, since it may not be obvious to users that they can implement the main "loop" so simply.